sdk-core/src/main/java/co/reachfive/identity/sdk/core/WebauthnClient.kt

```diff
@@ -149,7 +154,8 @@ internal class WebauthnAuthClient(
         reachFiveApi
             .createWebAuthnRegistrationOptions(
                 authToken.authHeader,
-                WebAuthnRegistrationRequest(origin, newFriendlyName)
+                WebAuthnRegistrationRequest(originWebauthn, newFriendlyName),
+                if (origin != null) mapOf("origin" to origin) else emptyMap()
```

Comment:
Won't we have a conflict between the new origin and that in WebAuthnRegistrationRequest?

Reply:
It would not be a problem because the origin defined in the context is valued from the request header first. However, the origin is useless for this method as there are no user event triggered in the endpoint. I remove it.